libs/deposits/src/lib/approve-notification.tsx

```diff
@@ -166,11 +171,19 @@ const ApprovalTxFeedback = ({
       ? Number(selectedAsset.source.chainId)
       : undefined;
 
-  const txLink = tx.txHash && (
-    <EtherscanLink sourceChainId={chainId} tx={tx.txHash}>
-      {t('View on Etherscan')}
-    </EtherscanLink>
-  );
+  const chain = getChainName(chainId);
+
+  const txLink =
+    tx.txHash &&
+    (chain.includes('Ethereum') ? (
```

Comment:
**nitpick (non-blocking)**:  Use a chain ID lookup would probably be more robust than relying on the string. Not worth changing as bridges are not being added regularly.